osx-cross.sh

```diff
@@ -7,23 +7,38 @@
 
 set -eu -o pipefail
 
-PKG_DEPS="patch xz-utils clang"
+PKG_DEPS="patch xz-utils clang llvm file"
 
-apt-get update -qq
-time apt-get install -y -q $PKG_DEPS
+time apt-get install -y -q --no-install-recommends $PKG_DEPS
+
+# NOTE: when changing version here, make sure to
+# also change OSX_CODENAME below to match
+OSX_SDK=MacOSX10.10.sdk
 
-OSX_SDK=MacOSX10.11.sdk
 OSX_CROSS_COMMIT=a9317c18a3a457ca0a657f08cc4d0d43c6cf8953
 OSXCROSS_PATH="/osxcross"
 
+LIBTOOL_VERSION=2.4.6
+OSX_CODENAME=yosemite
+
 echo "Cloning osxcross"
 time git clone https://github.com/tpoechtrager/osxcross.git $OSXCROSS_PATH
 cd $OSXCROSS_PATH
 git checkout -q $OSX_CROSS_COMMIT
 
 echo "Downloading OSX SDK"
-time curl -sSL https://s3.dockerproject.org/darwin/v2/${OSX_SDK}.tar.xz \
-    -o "${OSXCROSS_PATH}/tarballs/${OSX_SDK}.tar.xz"
+time curl -sSL \
+	https://www.dropbox.com/s/yfbesd249w10lpc/${OSX_SDK}.tar.xz \
+	-o "${OSXCROSS_PATH}/tarballs/${OSX_SDK}.tar.xz"
```

Comment:
Wondering if we need a checksum for this download

Reply:
makes sense